elastic4s-core/src/main/scala/com/sksamuel/elastic4s/SearchDsl.scala

```diff
@@ -128,7 +129,7 @@ case class RescoreDefinition(query: QueryDefinition) {
 }
 
 
-case class SearchDefinition(indexesTypes: IndexesAndTypes) {
+case class SearchDefinition(indexesTypes: IndexesAndTypes) extends DefinitionAttributeMinScore {
```

Comment:
Was not sure if you eventually wanted to reuse the `DefinitionAttribute`s in the` SearchDefinition` as well.